lib/destroyed_at.rb

```diff
@@ -39,6 +39,7 @@ def destroyed(time = nil)
 
   # Set an object's destroyed_at time.
   def destroy(timestamp = nil)
+    timestamp ||= @_pending_destroyed_at_timestamp
```

Comment:
I would like to keep the assignment of `timestamp` to one line; can we write this as follows (and then kill `timestamp ||= current_time_from_proper_timezone`):

```rb
timestamp ||= @_pending_destroyed_at_timestamp || current_time_from_proper_timezone
```